policy/README.md

```diff
@@ -8,36 +8,48 @@ This specification describes the digital relationship between _mobility as a ser
 
 ## Table of Contents
 
-- [General Information](#general-information)
-  - [Versioning](#versioning)
-  - [Update Frequency](#update-frequency)
-- [Background](#background)
-- [Distribution](#distribution)
-- [REST Endpoints](#rest-endpoints)
-  - [Policies](#policies)
-  - [Geographies](#geographies)
-  - [Requirements](#requirements)
-- [Flat Files](#flat-files)
-- [Schema](#schema)
-  - [Policy](#policy)
-  - [Rules](#rules)
-  - [Rule Types](#rule-types)
-  - [Rule Units](#rule-units)
-  - [Geography](#geography)
-  - [Rate Recurrences](#rate-recurrences)
-  - [Messages](#messages)
-  - [Value URL](#value-url)
-  - [Order of Operations](#order-of-operations)
-  - [Requirement](#requirement)
-    - [Update Frequency](#requirement-update-frequency)
-    - [Public Hosting](#public-hosting)
-    - [Version Tracking](#version-tracking)
-    - [Beta Limitations](#beta-limitations)
-    - [Format](#requirement-format)
-    - [Metadata](#requirement-metadata)
-    - [Programs](#requirement-programs)
-    - [Data Specs](#requirement-data-specs)
-    - [APIs](#requirement-apis)
+- [Mobility Data Specification: Policy](#mobility-data-specification-policy)
```

Comment:
Remove page title and reduce indent for other items

Reply:
Fixed in 5e23f0b